test.js

```diff
@@ -70,8 +70,8 @@ test('test .destroy', function (t) {
 })
 
 test('unsorted entry, sorted iterator', function (t) {
-  var db = new MemDOWN('foo'),
-    noop = function () {}
+  var db = new MemDOWN('foo')
+  var noop = function () {}
```

Comment:
`noop` could be local function and just re-used everywhere, but refactoring test code? Meh!

Reply:
Wait, which commit are you reviewing? I did this already 😄

Reply:
@vweevers I'm reviewing commit per commit, I just think it's easier for me, sorry for the confusion :)

Reply:
I'll wait a little :)

Reply:
Oh nevermind